runtime/oti/stackwalk.h

```diff
@@ -338,12 +338,12 @@ extern "C" {
 #define JIT_RESOLVE_PARM(parmNumber) (walkState->walkedEntryLocalStorage->jitGlobalStorageBase[jitArgumentRegisterNumbers[(parmNumber) - 1]])
 #define J9SW_JIT_STACK_SLOTS_USED_BY_CALL 0x0
 #define J9SW_POTENTIAL_SAVED_REGISTERS 0x20
-#define J9SW_REGISTER_MAP_MASK 0xFFFFFFFF
+#define J9SW_REGISTER_MAP_MASK 0xFFFF0000
 #define J9SW_JIT_FIRST_RESOLVE_PARM_REGISTER 0x0
 #undef  J9SW_JIT_LOOKUP_INTERFACE_RESOLVE_OFFSET_TO_SAVED_RECEIVER
 #undef  J9SW_JIT_VIRTUAL_METHOD_RESOLVE_OFFSET_TO_SAVED_RECEIVER
-#define J9SW_LOWEST_MEMORY_PRESERVED_REGISTER jit_r18
-#define J9SW_JIT_CALLEE_PRESERVED_SIZE 11
+#define J9SW_LOWEST_MEMORY_PRESERVED_REGISTER jit_r21
+#define J9SW_JIT_CALLEE_PRESERVED_SIZE 8
```

Comment:
I believe that `jitCalleeSavedRegisterList` must be changed so that it is consistent with this change.
https://github.com/eclipse/openj9/blob/07ea932b9e71583318f5fa9d1820faf0d2c93d61/runtime/util/jitregs.c#L388-L402

For example, the following loop currently iterates on only a part of `jitCalleeSavedRegisterList`.
https://github.com/eclipse/openj9/blob/07ea932b9e71583318f5fa9d1820faf0d2c93d61/runtime/codert_vm/jswalk.c#L1406-L1410

Reply:
It looks like `J9SFJ2IFrame` needs to be changed too.
https://github.com/eclipse/openj9/blob/faed575114585425c471b1eafb7b6220d2f1a9ff/runtime/oti/j9nonbuilder.h#L2586-L2598

Reply:
Done.  Fixed the range, although the ordering (high->low and low->high still needs to be sorted out definitively).